commons-rng-core/src/test/java/org/apache/commons/rng/core/source64/PcgRxsMXs64Test.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.commons.rng.core.source64;
+
+import org.apache.commons.rng.core.RandomAssert;
+import org.junit.Test;
+
+public class PcgRxsMXs64Test {
+    @Test
+    /*
+     * Tested with respect to pcg_engines::setseq_rxs_m_xs_64_64 of the C++ implementation.
+     * See : http://www.pcg-random.org/download.html#cpp-implementation
+     */
+    public void testReferenceCode() {
+        final long[] expectedSequence = {
+            0xc147f2291fa40ccfL, 0x8edbcbf8a5f49877L, 0x61e05a1d5213f0b4L, 0xc039f9369032e638L,
```

Comment:
A formatting nit. If your 16 character hex number is less that 16 chars then add leading zeros: `0x000...`

The same for the other units tests with 8 character hex numbers.

Reply:
Done.